### PR TITLE
Copy compiler args from compileKotlin to kspKotlin

### PR DESCRIPTION
**Pull request by ting-yuan**
Will add a test after #269 is merged.

### CODE REVIEW

File: gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspSubplugin.kt [hunk 1]
```diff
@@ -133,7 +141,10 @@ class KspGradleSubplugin @Inject internal constructor(private val registry: Tool
             kspTask.setDestinationDir(destinationDir)
             kspTask.mapClasspath { kotlinCompileProvider.get().classpath }
```
Comment (yigit) [inline gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspSubplugin.kt:142]: should we update this to also delegate in the task?

Comment (ting-yuan) [reply gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspSubplugin.kt]: Done. Thanks!

**Comment by chrisjenx** [comment]
Just wondering what your release cadence is? (looks like almost every weekish?) looking to try this fix out :)

**Comment by ting-yuan** [comment]
The release is generally bi-weekly and the next one will be on Feb.3.

**Review by dtdharan — APPROVED** [review]

**Pull request merged**